tests/test_scheduler_query.py

```
@@ -0,0 +1,113 @@
+        leaguepedia_id="Test Match 1",
+        contest_id=contest.id,
+        team1="Team A",
+        team2="Team B",
+        scheduled_time=match_time,
+    )
+    session.add(match)
+    await session.commit()
+
+    # Execute the function under test
+    # This should NOT raise NotImplementedError or any other exception
+    fetched_now, matches = await _get_matches_starting_soon(session)
```

Ruff (0.14.10)

Unpacked variable `fetched_now` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)